src/vs/workbench/contrib/terminal/terminalContribExports.ts

```diff
@@ -32,6 +32,7 @@ export const enum TerminalContribCommandId {
 export const enum TerminalContribSettingId {
 	SuggestEnabled = TerminalSuggestSettingId.Enabled,
 	StickyScrollEnabled = TerminalStickyScrollSettingId.Enabled,
+	SuggestProviders = TerminalSuggestSettingId.Providers
```

Comment:
We don't need this here, this const enum is only for the edge case where settings defined in contribs are used outside the contribs (which is bad).